tests/Entity/JsonPointerTest.php

```diff
@@ -92,4 +92,16 @@ public function getTestData()
 
         );
     }
+
+    public function testJsonPointerWithPropertyPaths()
+    {
+        $initial = new JsonPointer('#/definitions/date');
+        $modified = $initial->withPropertyPaths(array('~definitions/general', '%custom%'));
```

Comment:
@jojo1981 i'm fine adding those two lines - would you mind explaining why you think they are necessary? i can't get my head around, what are those lines asserting that the current test doesn't? it's rather a duplication of the existing tests - which obviously doesn't hurt, but it still doesn't relate to the question if the method returns a new object or modified the existing

if we want to have additional checks, what about [assertNotSame](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertSame) for `$initial` and `$modified` ?